lib/nexpose/global_blackout.rb

```diff
@@ -0,0 +1,47 @@
+module Nexpose
+
+  class GlobalBlackout < APIObject
+    include JsonSerializer
+
+    # [Array] Blackout starting dates, times and duration for blackout periods.
+    attr_accessor :blackout
+
+    def initialize(blackout)
+      @global_blackout = Array(blackout)
+    end
+
+    def save(nsc)
+      params = to_json
+      JSON.parse(AJAX.post(nsc, '/api/2.1/silo_blackout/', params, AJAX::CONTENT_TYPE::JSON))
+    end
+
+    def to_h
+      {
+          blackouts:
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.